### ISSUE TITLE
Default bindings use hardcoded env var syntax.

### ISSUE THREAD

**Issue by KaitlynEthylia** [gokcehan/lf#1254]
By default, pressing `e` on windows is set to run the command "%EDITOR% %f%" as a hardcoded string, instead of using `os.Getenv` (this applies also to the pager, opener, and shell bindings). this means that when using any shell other than CMD, these default bindings have to be rewritten as they don't recognise the %var% syntax. I think this should be changed to first resolve the variables and then execute the command, to prevent this hassle

**Comment by gokcehan** [comment]
@KaitlynEthylia I agree this is not ideal, but I'm not sure how this could be implemented with the current structure of the code. I think we can easily expand `%EDITOR%`  to its value but `%f%` itself changes at every execution so we would have to somehow redefine the command each time it is used. If someone can find a reasonable way to make this work without any dramatical changes, let us know and we can discuss it further. Otherwise, the recommended way is to use one of the example config files under `etc` directory appropriate for your choice of shell. Note, example config files already redefines these default command with the appropriate shell syntax to get around this issue.

**Comment by joelim-work** [comment]
While it is true that the default bindings (e.g. `e` for editor) is hardcoded to use CMD syntax, I don't think it is an issue because the default shell is also set to `cmd`. If you are going to configure `lf` to use a different shell like PowerShell, you can just simply redefine all the relevant keybindings as well. For convenience, the PowerShell equivalents are now defined in the example config file:

https://github.com/gokcehan/lf/blob/fe3a97c5d046cb9ee7de5bc981c5ff6d6125deef/etc/lfrc.ps1.example#L19-L28
